### DISCUSSION TITLE
General

### DISCUSSION THREAD

**Discussion by willwulfken** [willwulfken/MidJourney-Styles-and-Keywords-Reference#9] [General]
General Discussion

**Comment by odobostudio** [comment]
Hi there - amazing work - I wondered what was the original query you use for the standardizing of the all the images 

I would be interested in seeing the prompts used for the initial creation - if you are willing to share

thank you in advance

**Comment by willwulfken** [comment]
The general query should be under control image and seed on the homepage.

But either way, these are the prompt formats:

Sphere images:
- sphere, <style> --seed 4776

Landscape images (for artists):
- landscape, <style> --seed 4776

Just the style:
- <style> --seed 4776

For the different versions and such, I just added them before --seed 4776

**Comment by Atlari** [comment]
These tones of red and blue have been showing up way too much. There must be a way to diversify the hues of color the AI  comes up with. ![image](https://user-images.githubusercontent.com/113123962/189549960-03d90766-f5a1-4892-ab53-c659feb47f4e.jpeg)

**Reply by willwulfken** [reply]
This is a good suggestion! There are some in [[🎨 Palettes and Color Styles](https://github.com/willwulfken/MidJourney-Styles-and-Keywords-Reference/blob/main/Pages/MJ_V3/Style_Pages/Colors_Palettes_and_Color_Styles.md)] > [🎨🖌 Chromatic Palettes] but adding more would definitely be a good update to that

**Comment by altdann** [comment]
Just wanted to say, I found out about MidJourney on Thursday..? Haven't slept since. And now after Matt's youtube video, discovering this treasure of a library. Realizing that the sandman's not even close, and you! my good Sir, are a scripting God

**Reply by kamikush** [reply]
The imagination never sleeps

**Comment by Atlari** [comment]
What video?

**Comment by hot2hot** [comment]
Hello. I have generated image https://i.postimg.cc/j5BWrkYc/photo-2023-03-01-15-40-10.jpg
I need to make exactly the same, so that only the lion is red. How can I do that?

And then the picture is constantly changing completely. Help me please. I won't be in debt)
